code/modules/mining/machine_unloading.dm

```diff
@@ -28,10 +28,10 @@
 		var/turf/input = get_step(src, input_dir)
 		var/obj/structure/ore_box/BOX = locate() in input
 		if(BOX)
-			var/turf/output = get_step(src, output_dir)
 			var/i = 0
 			for(var/obj/item/ore/O in BOX.contents)
-				O.forceMove(output)
+				contents -= O
+				O.loc = src.output_dir
```

Comment:
That won't work because `output_dir` is a number, not a turf.
This one would work if you kept `output` and used that instead